package.json

```diff
@@ -67,13 +67,13 @@
     "@sindresorhus/slugify": "^2.2.1",
     "@stripe/stripe-js": "^4.1.0",
     "@t3-oss/env-nextjs": "^0.10.1",
-    "@tanstack/react-query": "^4.36.1",
+    "@tanstack/react-query": "^5.53.3",
     "@tanstack/react-table": "^8.20.1",
     "@tremor/react": "^3.17.4",
-    "@trpc/client": "^10.43.6",
-    "@trpc/next": "^10.45.2",
```

Comment:
upgraded to trpc 11 to useSuspenseQuery from "@tanstack/react-query". useSuspenseQuery is introduced in `"@tanstack/react-query` v5 upgrading only this package breaks trpc